README.md

```diff
@@ -31,6 +33,37 @@ module Robots
 end
 ```
 
+By default, the druid will be set to the completed state, but you can optionally have it set to skipped by creating a ReturnState object as shown below
+```ruby
+module Robots
+  module DorRepo
+    module Accession
+
+      class Shelve
+        include LyberCore::Robot
+
+        def initialize
+          super('dor', 'accessionWF', 'shelve')
+        end
+
+        def perform druid
+          obj = Dor::Item.find(druid)
+          if some_logic_here_to_determine_if_shelving_occurs
+            obj.shelve
+            return LyberCore::Robot::ReturnState.new('completed') # set the final state to completed
+          else
+            # just return skipped if we did nothing
+            return LyberCore::Robot::ReturnState.new('skipped') # set the final state to skipped
```

Comment:
If all we're going to pass back is the status right now, then it might be cleaner to have the caller just reference a class constant like so:

```ruby
  return LyberCore::Robot::ReturnState.SKIPPED
```

and have that thing be an instance of ReturnState

```ruby
LyberCore::Robot::ReturnState.SKIPPED = LyberCore::Robot::ReturnState.new('skipped')
```